examples/kubernetes/pre-provision/preprov-pv.yaml

```diff
@@ -14,9 +14,8 @@ spec:
   volumeMode: "Filesystem"
   csi:
     driver: "filestore.csi.storage.gke.io"
+    # Modify this to use the one, filestore instance and share name
     volumeHandle: "modeInstance/us-central1-c/test-preprov-fs/vol1"
-    fsType: "nfs"
     volumeAttributes:
-      ip: 10.175.240.186 # MODIFY THIS TO Pre-provisioned FS INSTANCE IP
-      volume: vol1
-
+      ip: 192.168.61.42 # Modify this to Pre-provisioned Filestore instance IP
```

Comment:
Maybe change the IP to XXX.XXX.XX.XX to make it obvious that it has to be changed (and maybe even fail validation if it's not changed?)

Reply:
Changed it. Can you elaborate on the validation part?

Reply:
I mean that if the user forgets to update the IP maybe the resource will fail to create because it's an invalid IP.

Nothing we have to do here, sorry for the confusion.